src/components/Signup/Signup.jsx

```diff
@@ -62,6 +64,7 @@ export default function Signup() {
                         Username:
                         <input
                             name='name'
+                            value={values.name || ""}
```

Comment:
this is the correct variant with `value`.  But I see everywhere `defaultValue`. You need to use `value` everywhere because `defaultValue` is used only with `useRef`. For controlled components you need to use only `value`